homeassistant/components/cover/template.py

```diff
@@ -369,9 +386,12 @@ def async_update(self):
             except ValueError as ex:
                 _LOGGER.error(ex)
                 self._tilt_value = None
-        if self._icon_template is not None:
+
+        icon_template = (self._icon_template if self._icon_template else
+                         self._entity_picture_template)
+        if icon_template is not None:
```

Comment:
This is wrong. The `icon_template` can now be an entity picture template too ? And then it renders to icon property? That makes no sense